Doc/regulatoryframework

AnnKaCatX opened the pull request:
This PR adds more information about the regulatory framework.

AnnKaCatX committed:
addcontent

AnnKaCatX committed:
change

AnnKaCatX committed:
change

AnnKaCatX committed:
fix linter issues

AnnKaCatX commented:
@werner-roman: I also adopted some content for the regulatory framework in the operating model. Can we keep it here in this Pull-request? Or should I revert the changes? I don't have access to the operating model repository.

AnnKaCatX committed:
fix mardownlinter issue

AnnKaCatX committed:
add picture

werner-roman commented:
> @werner-roman: I also adopted some content for the regulatory framework in the operating model. Can we keep it here in this Pull-request? Or should I revert the changes? I don't have access to the operating model repository.

Hey @AnnKaCatX - thanks a lot for the PR and input! :)
i would revert the changes in the OM as they must be added in the OM Repo (only accessible for the Committee)

cc: @danielmiehle

werner-roman commented:
@AnnKaCatX  - here would be also the flight level image available in higher quality: https://github.com/catenax-eV/catenax-ev.github.io/blob/main/docs/operating-model/how-data-space-governance/assets/catena-x-governance-framework-flight-levels.png

werner-roman commented on docs/operating-model/how-data-space-governance/assets/flightlevels.png:
Will taken care of in OM

werner-roman commented on docs/operating-model/how-data-space-governance/how-data-space-governance.md:
Updated in OM

werner-roman commented on docs/regulatory-framework/governance-framework.md:
```diff
@@ -17,22 +19,35 @@ To easily grasp the layers of our Governance Framework, think of flight level as
+At this level, the governance framework is defined by the Catena-X Association and the basic principles of the data ecosystem are outlined. Compliance is mandatory for all participants in the data space.
+There are several normative documents to consider.
+
+The first is the operating model. The operating model lays the foundation and is the pre-requisite for further standardization, certification, implementation, and operations of software components and partners in the Catena-X data space. The operating model is continuously updated to reflect the evolution of business and regulatory requirements, as well as emerging technologies.
```
add link to OM?

AnnKaCatX replied:
Done

werner-roman commented on docs/regulatory-framework/governance-framework.md:
```diff
@@ -17,22 +19,35 @@ To easily grasp the layers of our Governance Framework, think of flight level as
+
+The first is the operating model. The operating model lays the foundation and is the pre-requisite for further standardization, certification, implementation, and operations of software components and partners in the Catena-X data space. The operating model is continuously updated to reflect the evolution of business and regulatory requirements, as well as emerging technologies.
+
+The "10 Golden Rules" document, which outlines the basic principles of the data ecosystem, is published by the Catena-X association and provides clear guidelines for necessary specifications at lower levels. It also specifies how far a technical standard can go. The agreement between the CSPs and participants (e.g., data providers and data consumers) is established during onboarding.
```
link to 10 golden rules?

AnnKaCatX replied:
Done

Commit:
fixed link + spelling

werner-roman commented on docs/regulatory-framework/governance-framework.md:
```diff
@@ -17,22 +19,35 @@ To easily grasp the layers of our Governance Framework, think of flight level as
+
+The third document is the Data Exchange Governance. This Data Exchange Governance sets out the key principles for data exchange that apply to each Participant when exchanging data in the Catena-X Data Space for one or more purposes as predefined by the Association ("Predefined Purpose(s)"). Each Participant must agree to this Data Exchange Governance in order to effect data exchanges through a re ...
 
 ## 20,000ft - Use Case Level
```
The rework of 20,000 is missing - there are still all the Use-Case Framework agreements: 
![image](https://github.com/user-attachments/assets/82d76d04-4252-4b8f-bcc7-7e4e758a49b4)

jSchuetz88 replied:
Will there still be content for 20,000ft after that or will this section be empty?

werner-roman commented on docs/regulatory-framework/governance-framework.md:
```diff
@@ -17,22 +19,35 @@ To easily grasp the layers of our Governance Framework, think of flight level as
+At this level, the Catena-X Association provides specific guidelines for each use case. The basic requirements for data usage in a use case are outlined. Participants are required to confirm compliance with the standards and policies in order to participate in a specific use case. This is an essential and unique part of our Catena-X data space and allows our use cases to scale across the network  ...
+
+The standardised usage policies represent a consolidation of the predefined use case frameworks. The only difference is the purpose of use.
+The Catena-X ODRL profile is used for this purpose. The primary aim of the Catena-X ODRL profile is to provide standardised contract modules such as permissions, prohibitions and obligations in a machine-readable format that can be checked automatically. The ODRL profile enables the automation of contract negotiations in the Catena-X data room, as contract negotiations are conducted electronicall ...
```
data room - missspell -> data space?

werner-roman commented on docs/regulatory-framework/governance-framework.md:
```diff
@@ -17,22 +19,35 @@ To easily grasp the layers of our Governance Framework, think of flight level as
+While Data Providers and Data Consumers have the option of concluding bilateral contracts (supplier contract, ...) outside of a registered connector to determine their contractual relationship for data exchange via Catena-X, the Catena-X Regulatory Framework sets out certain boundary conditions (10 Golden Rules, Data Exchange Governance...) that are binding for all data space participants and can ...
 
-- "Digital behavior twin".
+These boundary conditions, combined with the possibility to automate data exchange contract negotiations, form one of the key benefits and foundations for scaling of the Catena-X Data Space. For more information on how to conclude data exchange contracts via Catena-X, please refer to the Catena-X memorandum on how to conclude data exchange contracts.
```
link if we say "for more information"

werner-roman reviewed (changes requested):
Thanks for the inital input.
the 20,000 must be reworked as there are still the old documents shown + some comments for the changes

jSchuetz88 reviewed (commented):
General question: Previously we had one document per use case and their AgreementCredendials. I would somehow expect that these will now be consolidated into a general document / agreements at the 30,000ft level. Will there be a separate document / file for that or is thiscovered by the general description of the regulatory framework?

danielmiehle commented on docs/operating-model/how-data-space-governance/how-data-space-governance.md:
```diff
@@ -185,26 +185,32 @@ Different from Catena-X Certification, Catena-X qualification is a label held by
+The Operating Model, the 10 Golden Rules and the Data Exchange Governance fall under the 30,000-ft level (Data Space Level) and are thus mandatory for all data space participants. At this level, the basic principles of the data ecosystem are outlined.
 
-Each higher level cascades into the lower ones, and the lower levels align with those above. Each level comes with specific guidelines and resources but also responsibilities for participants in our data space. This Operating Model falls under the 30,000 ft level and is thus mandatory for all data space participants. Maintaining and updating our Regulatory Framework for data space operations lies ...
+The standardised usage policies are located at the 20,000-ft level (Use Case Level), where the fundamental requirements for data usage in a use case are defined.
```
Ich dachte die use case policies werden konsolidiert zu 1 "data exchange governance" policy?
Falls das der Fall ist, liegt diese weiterhin auf 20.000 FT oder 30.000 FT siehe Anmerkung oben.

danielmiehle replied:
Mein letzter Abstimmungsstand mit Julia, ersetzt das Portal gerade auch "nur" die Use Case Policies mit dem "data exchange governace" policy.

danielmiehle commented on docs/operating-model/how-data-space-governance/how-data-space-governance.md:
```diff
@@ -185,26 +185,32 @@ Different from Catena-X Certification, Catena-X qualification is a label held by
 
-![Catena-X Governance Framework flight levels](./assets/catena-x-governance-framework-flight-levels.png)
-*Catena-X Governance Framework flight levels*
+The Operating Model, the 10 Golden Rules and the Data Exchange Governance fall under the 30,000-ft level (Data Space Level) and are thus mandatory for all data space participants. At this level, the basic principles of the data ecosystem are outlined.
```
Data Exchange Governance auf 30.000 oder 20.000 ft?

danielmiehle commented on docs/operating-model/how-data-space-governance/how-data-space-governance.md:
```diff
@@ -185,26 +185,32 @@ Different from Catena-X Certification, Catena-X qualification is a label held by
+The standardised usage policies are located at the 20,000-ft level (Use Case Level), where the fundamental requirements for data usage in a use case are defined.
 
-### Concluding Data Exchange Contracts via Catena-X
+The 10,000-ft level (Data Offering Level) provides guidance for data offerings by data providers / consumers.
```
Sind das dann weiterhin Papierverträge, oder? Könnte wir ein wenig konkretisieren...

danielmiehle commented on docs/operating-model/how-data-space-governance/how-data-space-governance.md:
```diff
@@ -185,26 +185,32 @@ Different from Catena-X Certification, Catena-X qualification is a label held by
 ### Standardized modules for data exchange contracts
 
-The Catena-X Association aims to develop further automation and scalability of the process of concluding data exchange contracts via Catena-X. Connectors will enable Participants to choose from a range of modules when negotiating and agreeing on terms within the Data Exchange Governance (e.g. liability/dispute resolution/choice of law etc.). All official modules provided by the Catena-X Associati ...
+The Catena-X Association aims to develop further automation and scalability of the process of concluding data exchange contracts via Catena-X. Connectors will enable Participants to choose from a range of modules when negotiating and agreeing on terms within the Data Exchange Governance (e.g. liability/dispute resolution/choice of law etc.). All official modules provided by the Catena-X Associati ...
```
typo: Participants --> klein schreiben...?

danielmiehle commented on docs/operating-model/how-data-space-governance/how-data-space-governance.md:
```diff
@@ -185,26 +185,32 @@ Different from Catena-X Certification, Catena-X qualification is a label held by
-As of Release 24.05., these modules are limited to standardized data usage policies for Catena-X use cases.
+### Concluding Data Exchange Contracts via Catena-X
+
+The Catena-X Data Space enables Data Providers and Data Consumers to conclude data exchange contracts via registered connectors.
```
typo: data provider / consumer: Immer groß oder immer klein... 
@werner-roman : haben wir eine Regel wie wir mit Rollen Namen umgehen?

AnnKaCatX committed:
deleteusecaseandadddataexchange

AnnKaCatX committed:
Listchange

AnnKaCatX committed:
change_dataexchangegovernance_to_20,000ft-level

Commit:
# change files to ".md"
# add chatper informations
# added disclaimer in admonitions

AnnKaCatX committed:
delete_use_case_frameworks

AnnKaCatX committed:
change_headline_summarizing_and_add_changelog

Commit:
# corrected Changelog added 1.0.0 as well
# change version to 2.0.0 as this is a breaking change

AnnKaCatX committed:
change_category_headliner

Commit:
# corrected changelog removed duplicates
+ added link to changelog which points to the raised PR
# used links instead of a copy of the url
+ used admonitions

werner-roman commented on docs/regulatory-framework/20000ft/Standardized Usage Policies.md:
```diff
@@ -0,0 +1,11 @@
+
+The standardized usage policies represent a consolidation of the predefined use case frameworks. The only difference is the purpose of use.
+
+The Catena-X ODRL profile is used for this purpose. The primary aim of the Catena-X ODRL profile is to provide standardised contract modules such as permissions, prohibitions and obligations in a machine-readable format that can be checked automatically. The ODRL profile enables the automation of contract negotiations in the Catena-X data room, as contract negotiations are conducted electronicall ...
```
Data space not data room

werner-roman reviewed (changes requested):
spelling

werner-roman commented on docs/regulatory-framework/governance-framework.md:
```diff
@@ -17,22 +19,43 @@ To easily grasp the layers of our Governance Framework, think of flight level as
+:::
+
+The standardized usage policies represent a consolidation of the predefined use case frameworks. The only difference is the purpose of use.
+The Catena-X ODRL profile is used for this purpose. The primary aim of the Catena-X ODRL profile is to provide standardised contract modules such as permissions, prohibitions and obligations in a machine-readable format that can be checked automatically. The ODRL profile enables the automation of contract negotiations in the Catena-X data room, as contract negotiations are conducted electronicall ...
```
Data space not data room

AnnKaCatX committed:
changed_data_room_to_data_space

AnnKaCatX committed:
changed_data_room_to_data_space

werner-roman reviewed (approved):
LGTM 🚀 

@AnnKaCatX  - please align with @jSchuetz88 / @stephanbcbauer  when to merge an release! :)

jSchuetz88 commented on docs/regulatory-framework/changelog.md:
```diff
@@ -9,18 +9,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-- Moved China from Yellow List to White List, allowing onboarding of Chinese companies to the Catena-X network
-- Changed White List to Allow List and Yellow List to Conditional List
+- Changed White List to Allow List and Yellow List to Conditional List [[#74](https://url.de.m.mimecastprotect.com/s/sobUCMZkL4hqQZ2lYuwfgsQj2J-?domain=github.com)]
```
Ist dass sinnvoll hier auf einen PR zu verweisen? Vor allem mit einem Merkwürdigen secure link?

jSchuetz88 commented on docs/regulatory-framework/changelog.md:
```diff
@@ -9,18 +9,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
-- Changed White List to Allow List and Yellow List to Conditional List
+- Changed White List to Allow List and Yellow List to Conditional List [[#74](https://url.de.m.mimecastprotect.com/s/sobUCMZkL4hqQZ2lYuwfgsQj2J-?domain=github.com)]
 
 ### Removed
```
Wieso sind bei removed nur 5 Use Case Frameworks?
- PCF Exchange
- Quality
- Traceability

fehlen. Siehe suggestion

jSchuetz88 commented on docs/regulatory-framework/changelog.md:
```diff
@@ -5,11 +5,32 @@ sidebar_position: 5
+- Predefined Use Case Framework Business Partner Data Management V1.0 (valid until October 2024)
+- Predefined Use Case Framework Circular Economy V1.0 (valid until October 2024)
+- Predefined Use Case Framework PURIS V1.0 (valid until October 2024)
+
```
```suggestion
- Predefined Use Case Framework PCF Exchange V1.0 (valid until October 2024)
- Predefined Use Case Framework Quality V1.0 (valid until October 2024)
- Predefined Use Case Framework Traceability V1.0 (valid until October 2024)
```

jSchuetz88 commented on docs/regulatory-framework/30000ft/country-clearance-list.md:
```diff
@@ -3,9 +3,9 @@ pagination_prev: null
+# Catena-X: Country Clearance List
 
-## Clearance List "White List"
+## Clearance List "Allow List"
```
Ich weiß, dass hier abgestimmt wurde, dass Allow List genommen werden kann, wurde den geklärt, dass es in anderen Dokumenten aber dann auch umbenannt werden müsste?

jSchuetz88 reviewed (approved):
Some minor questions open for discussion, but nothin that prevents this PR.

Thank you all.

jSchuetz88 committed:
Update docs/regulatory-framework/changelog.md

jSchuetz88 committed:
Fix linting issue
